test/e2e/kubernetes/workloads/hostport-test.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: hostport-test
+spec:
+  completions: 1
+  backoffLimit: 0
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - image: curlimages/curl

```

Comment:
@jchauncey @sozercan do we have a local (MCR/ACR) image that we can use for this curl test, so that we can be better internet citizens?

Reply:
we have `mcr.microsoft.com/oss/tutum/curl:latest` in MCR but it's pretty old

Reply:
I think that's a better alternative than pounding dockerhub, thanks!